<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Initialize</class>
 <widget class="QMainWindow" name="Initialize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>339</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>301</width>
      <height>300</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Allowed Approximation Error (%)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="range_le">
       <property name="text">
        <string>0.1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Gravitational Acceleration</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="g_le">
       <property name="text">
        <string>9.81</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Density of Liquid</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Dynamic Viscosity of Liquid</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="mu_le">
       <property name="text">
        <string>1.002e-3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Approximate Reynold's Number</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="Re_le">
       <property name="text">
        <string>3500000</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Approximate Flow</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="qa_le">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Number of Elements</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="E_le"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Number of Nodes</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="N_le"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="ro_le">
       <property name="text">
        <string>1000</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Dtl_Inpt_pb">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Go to Detail Input</string>
    </property>
   </widget>
   <widget class="QPushButton" name="opn_fle_pb">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>range_le</tabstop>
  <tabstop>g_le</tabstop>
  <tabstop>ro_le</tabstop>
  <tabstop>mu_le</tabstop>
  <tabstop>Re_le</tabstop>
  <tabstop>qa_le</tabstop>
  <tabstop>E_le</tabstop>
  <tabstop>N_le</tabstop>
  <tabstop>Dtl_Inpt_pb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
